doc/tutorials/mlip/mlip.ipynb

```diff
@@ -0,0 +1,722 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vis.zndraw.selection = [305, 306]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To run the MD we will switch to a Velocity Verlet integrator and set the thermostat to 400 K."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "system.integrator.set_vv()\n",
+    "system.thermostat.set_langevin(kT=(400 * ureg.K * ureg.boltzmann_constant).m_as(\"eV\"), gamma=2, seed=42)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Conceptually the process for the MD is the same with the geometry optimization. However, this time we only show every 5th frame to the visualizer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tbar = tqdm.trange(500, ncols=120)\n",
+    "for idx in tbar:\n",
+    "    atoms = system.ase.get()\n",
+    "    atoms.calc = mace_mp_calc\n",
+    "    system.part.all().ext_force = atoms.get_forces()\n",
+    "    if idx % 5 == 0:\n",
+    "        vis.zndraw.append(atoms)\n",
+    "    system.integrator.run(1)\n",
+    "    tbar.set_description(f\"E_pot: {atoms.get_potential_energy():.3f} eV\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A good indicator to see the protonation events is the distance between the hydrogen and the oxygen atom in the hydroxyl group of the sulfuric acid.\n",
+    "We calculate these distances for each frame in the visualization together with the potential energy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "OH_1 = []\n",
+    "OH_2 = []\n",
+    "energies = []\n",
+    "for atoms in vis.zndraw:\n",
+    "    OH_1.append(atoms.get_distance(300, 305, mic=True))\n",
+    "    OH_2.append(atoms.get_distance(304, 306, mic=True))\n",
+    "    energies.append(atoms.get_potential_energy())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(10, 6))\n",
+    "plt.plot(OH_1, label=\"OH1\")\n",
+    "plt.plot(OH_2, label=\"OH2\")\n",
+    "plt.xlabel(\"Frame\")\n",
+    "plt.ylabel(\"O$-$H bond distance (Å)\")\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Exercise:**\n",
+    "- Indentify the highlighted atoms in the visualization window, which are the two hydrogen atoms of the sulfuric acid.\n",
+    "- Observe the moment they undergo the protonation event visually and also in the hydrogen-oxygen distance plots.\n",
+    "- Can you also see changes in the potential energy? Why is that?"
```

Comment:
It's unclear to me, what is the answer to this question. The dip in energy could be due to the exothermic reaction $\mathrm{H_2SO_4 + H_2O \to HSO_4^{-} + H_3O^{+}}$, but the energy difference doesn't reproduce experimentally determined Gibbs free energy values. I'm also unsure if the potential energy can be directly equated to a Gibbs free energy, a Helmholtz free energy,  an enthalpy or an internal energy.

Reply:
I would actually like to discuss this point during the tutorial. The question of whether this is a physically realistic process is fundamental to the explainability of ML potentials. During the discussion, I would also ask how to extend the study to determine the answer. Namely, running it longer and generating some statistics or pH measurements that can be compared to experimental values.